test/elastic_record/relation/batches_test.rb

```diff
@@ -16,6 +16,15 @@ def setup
   #   assert_equal [@red_widget, @blue_widget, @green_widget].to_set, results.to_set
   # end
 
+  def test_find_hits_in_batches
+    results = []
+    Widget.elastic_relation.find_hits_in_batches do |hits|
+      results << hits
+    end
+    expected_ids = [[@red_widget, @blue_widget, @green_widget].to_set]
```

Comment:
These are `SearchHit` objects, right? Not ids.
```suggestion
    expected_hits = [[@red_widget, @blue_widget, @green_widget].to_set]
```